js/src/interface.js

```diff
@@ -274,6 +267,29 @@ let app = new Vue({
     }
   },
   methods: {
+    initDataSourceProvider(currentDataSourceId) {
+      let dataSourceData = {
+        dataSourceTitle: settings.dataSourceTitle ? settings.dataSourceTitle : 'Select data source',
+        dataSourceId: currentDataSourceId,
+        appId: Fliplet.Env.get('appId'),
+        default: _.extend({
+          name: `Data for ${Fliplet.Env.get('appName')}`,
```

Comment:
@AndrRyaz I just found out this code is using ES6, please change it to ES5 as it's not supported on IE11.

Reply:
@squallstar 
Nick, I want to clarify that this component is written in vue and is built using gulp so it is possible to use es6

Reply:
@AndrRyaz you're right, sorry I have missed that in this component: 👍🏻 

Once we have the automatic ESLint plugged in on GitHub we won't have to keep verifying these.